README.md

```diff
@@ -1,49 +1,61 @@
-# Jujutsu VCS
+<div class="title-block" style="text-align: center;" align="center">
 
-![](https://img.shields.io/github/license/martinvonz/jj) ![](https://img.shields.io/github/v/release/martinvonz/jj) ![](https://img.shields.io/github/release-date/martinvonz/jj) ![](https://img.shields.io/crates/v/jj-cli)
-<br/>
-![](https://github.com/martinvonz/jj/workflows/build/badge.svg) ![](https://img.shields.io/codefactor/grade/github/martinvonz/jj/main) ![](https://img.shields.io/librariesio/github/martinvonz/jj)
+# Jujutsu &mdash; a version control system
 
-- [Disclaimer](#disclaimer)
-- [Introduction](#introduction)
-- [Getting started](#getting-started)
-- [Status](#status)
-- [Related work](#related-work)
+![](https://img.shields.io/github/license/martinvonz/jj)
+![](https://img.shields.io/github/v/release/martinvonz/jj)
+![](https://img.shields.io/github/release-date/martinvonz/jj)
+![](https://img.shields.io/crates/v/jj-cli)
+<br/> ![](https://github.com/martinvonz/jj/workflows/build/badge.svg)
+![](https://img.shields.io/codefactor/grade/github/martinvonz/jj/main)
+![](https://img.shields.io/librariesio/github/martinvonz/jj)
 
-## Disclaimer
+<strong>
+  <a href="https://martinvonz.github.io/jj">Homepage</a>
+  &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+  <a href="https://martinvonz.github.io/jj/v0.9.0/install-and-setup.html">Installation</a>
+  &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+  <a href="https://martinvonz.github.io/jj/v0.9.0/tutorial.html">Getting Started
+  Tutorial</a>
+</strong>
```

Comment:
Can we not achieve similar formatting with markdown?

Reply:
I've reworked this a bit; let me know if you like it better...